code/modules/mob/dead/observer/observer.dm

```diff
@@ -657,6 +657,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		return TRUE
 
 	return FALSE
+	
+/mob/dead/observer/incapacitated()
+	return TRUE

```

Comment:
All objects (and as such machines) share the same Topic check system with NanoUI.